internal/server/iigointernal/sanctionevaluator.go

```diff
@@ -7,6 +7,24 @@ import (
 	"gonum.org/v1/gonum/mat"
 )
 
+// DefaultInitLocalSanctionCache generates a blank sanction cache
+func DefaultInitLocalSanctionCache(depth int) map[int][]shared.Sanction {
```

Comment:
Why move and export these if we aren't using them in any other package?

Reply:
AAAAAA

Reply:
Nice